moa-quickstart.md

```diff
@@ -0,0 +1,338 @@
+# MOA quickstart guide
+
+This guide walks through setting up your first MOA cluster using `moactl`.
+
+This includes the following steps:
+
+* Installing prerequisites
+* Creating your cluster
+* Adding users to your cluster
+* Cleaning up
+
+By the end of this guide you will have a Amazon Red Hat OpenShift cluster running in your AWS account.
+
+## Installing prerequisites
+
+Complete the following prerequisites before creating your MOA cluster.
+
+### Install the required binaries
```

Comment:
The init command tells you where to download the oc CLI, so I would say move this section below init, or strike it entirely.

Reply:
Good idea. I removed this section and moved the line on installing the aws cli to the section below.